pages/index.js

```diff
@@ -67,6 +70,7 @@ export default function Home() {
           {seconds} seconds, <br />
           🍆
         </h1>
+        <Image src={teduardoImage} alt="Teduardo" width={1000} height={1333} />
```

Comment:
The `{}` syntax is when you need to reference a variable. If you're not referencing a variable you can just do "" like below:
```react
<Image src={teduardoImage} alt="Teduardo" width="1000" height="1333" />
```
BUT if you reference a static height and width then the image won't resize automatically, so I think you can just not use a width and height and it will resize automatically.